udata_tabular_preview/preview.py

```diff
@@ -32,5 +33,9 @@ def can_preview(self, resource):
 
         return all((has_config, is_supported, is_allowed, size_ok))
 
+    @property
+    def preview_base_url(self):
```

Comment:
I don't think we need `serve_url` anymore? We should use `preview_base_url` in `can_preview` instead, right?

Reply:
`TABULAR_CSVAPI_URL` is required for the `explore.vue` component but we can remove the `@property`